### PR TITLE
level2 file batch processing

### LINKED ISSUE #47: Automate NC-processing
Blocked by: #45

### PR DESCRIPTION
**Pull request by BengtRydberg**
adding script that will allow for an automatic batch processing of monthly level2 files.

the intention is that we will run this script from a crontab on malachite inside a docker container.

something like this: 
docker run --rm -v ~/.odin_level2.json:/odin_level2.json:ro python odinapi/utils/batch_smrl2filewriter.py odin_level2.json /data/odin-l2-data

the odin_level2.json file specify what product files we will generate, the included test file is such an example.

Existing level2 files will not be overwritten with this processing.

Resolves #47

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,6 +6,7 @@ skipsdist = true
 deps=
 	pytest
 	pytest-docker-db
+        pytest-freezegun
```
Comment (local-minimum) [inline tox.ini:9]: looks like different indentation compared to the other?

**Review by local-minimum — COMMENTED** [review]
lgtm, but perhaps better if @skymandr  or @joakimmoller checks it that understands odin processing better than me.

File: tox.ini (hunk 1 above)
Comment (BengtRydberg) [reply tox.ini]: I missed that the other ones were tabbed, it still looks odd, I will fix this. should it be tabs?

Comment (local-minimum) [reply tox.ini]: I think spaces are nicer

File: src/test/fixtures/odinl2.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (skymandr) [inline src/test/fixtures/odinl2.json:1]: Kan vi använda denna för #37 ?

**Review by skymandr — APPROVED** [review]
lgtm

**Pull request merged**